README.md

```diff
@@ -5,10 +5,8 @@ JMDict is a simple library written in Go for parsing the raw data files for the
 [JMnedict](http://www.edrdg.org/enamdict/enamdict_doc.html) (names), and
 [KANJIDIC](http://nihongo.monash.edu/kanjidic2/index.html) (Kanji) dictionaries. As far as I know, these are the only
 publicly available Japanese dictionaries and are therefore used by a variety of tools (including
-[Yomichan-Import](https://foosoft.net/projects/yomichan-import) from this site).
+[Yomitan-Import](https://github.com/themoeway/yomitan-import) from this site).
 
 The XML format used to store dictionary entries and entity data was deceptively annoying to work with, leading to the
-creation of this library. Please see the [documentation page](https://godoc.org/foosoft.net/projects/jmdict) for a
+creation of this library. Please see the [documentation page](https://godoc.org/github.com/themoeway/jmdict-go) for a
```

Comment:
It seems like this redirects to this link now: https://pkg.go.dev/foosoft.net/projects/jmdict

See here for what I found about it: https://go.dev/blog/godoc.org-redirect

Reply:
Resolved in https://github.com/themoeway/jmdict-go/pull/1/commits/706ae7a88d3900556c0b1d9734b289f658e1086f